apps/synapse-portal-framework/src/components/ComponentCollapse.tsx

```diff
@@ -1,28 +1,25 @@
-import { SynapseConfig } from '../types/portal-config'
-import React, { useState } from 'react'
-import { Box, Collapse, SxProps, Typography } from '@mui/material'
 import {
   KeyboardArrowDownTwoTone,
   KeyboardArrowUpTwoTone,
 } from '@mui/icons-material'
-import { SynapseComponent } from '../components/SynapseComponent'
+import { Box, Collapse, SxProps, Typography } from '@mui/material'
+import React, { useState } from 'react'
 
-export type SynapseComponentCollapseProps = {
+export type ComponentCollapseProps = React.PropsWithChildren<{
   text: string
-  synapseConfig: SynapseConfig
```

Comment:
A common theme in this changeset: instead of passing a `synapseConfig`, just render a ReactNode and pass it in (e.g. as a child).